code/controllers/subsystem/mapping.dm

```diff
@@ -95,10 +94,6 @@ SUBSYSTEM_DEF(mapping)
 
 #define INIT_ANNOUNCE(X) to_chat(world, "<span class='boldannounce'>[X]</span>"); log_world(X)
 
-/datum/controller/subsystem/mapping/proc/run_map_generation()
-	for(var/area/A in world)
```

Comment:
This was seriously a var in world loop? 
Those are notorious for causing crashes who coded this